lightning/src/ln/functional_tests.rs

```diff
@@ -2334,24 +2334,32 @@ fn claim_htlc_outputs_single_tx() {
 		// ChannelMonitor: local commitment + local HTLC-timeout (2)
 
 		// Check the pair local commitment and HTLC-timeout broadcast due to HTLC expiration
+		assert_eq!(node_txn[3].input.len(), 1);
+		check_spends!(node_txn[3], chan_1.3.clone());
```

Comment:
Same here - can you do a pass for clone() in check_spends? Seems like you reintroduced a bunch of them.